api/app/Http/Controllers/Api/MailController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use Mail;
+use App\Mail\PasswordReset;
+use App\Http\Controllers\Controller;
+
+class MailController extends Controller
```

Comment:
Why creating a MailController for sending a sendPasswordResetMail?
I'd suggest to do it in the PasswordResetController or create a separate service